generator/features/__snapshots__/assetListing.spec.ts.snap

```diff
@@ -244,10 +244,14 @@ contract CreateProposal is EthereumScript {
     PayloadsControllerUtils.Payload[] memory payloads = new PayloadsControllerUtils.Payload[](1);
 
     // compose actions for validation
-    IPayloadsControllerCore.ExecutionAction[]
-      memory actionsEthereum = new IPayloadsControllerCore.ExecutionAction[](1);
-    actionsEthereum[0] = GovV3Helpers.buildAction(type(AaveV3Ethereum_Test_20231023).creationCode);
-    payloads[0] = GovV3Helpers.buildMainnetPayload(vm, actionsEthereum);
+    {
```

Comment:
scoping is now needed as otherwise we get stack to deep when deploying a proposal on all networks.